src/lib/Constants.ts

```diff
@@ -64,6 +64,7 @@ export const DataType = {
 
 /** CSS class and id constants. */
 export const CSS = {
+  // TODO(cjcox): Refactor out these MDL classes in favor of flexbox.
   MDL_LIST:               "mdl-list",
```

Comment:
It's Google Style to prefer single quotes.

I was in the habit of doing doubles before working here, but have switched to singles after reading that.  Ultimately, it's more important to be consistent within your project than adhere to that style guide, but being consistent (both within Material and with Google as a whole) is worth some consideration too.  😃